common/buildcraft/transport/stripes/StripesHandlerPipeWires.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2017 SpaceToad and the BuildCraft team
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+ */
+
+package buildcraft.transport.stripes;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.EnumDyeColor;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import buildcraft.api.transport.IStripesActivator;
+import buildcraft.api.transport.IStripesHandlerItem;
+import buildcraft.api.transport.pipe.IPipeHolder;
+import buildcraft.api.transport.pipe.PipeApi;
+
+public class StripesHandlerPipeWires implements IStripesHandlerItem {
+
+    private static final int PIPES_TO_TRY = 8;
+
+    @Override
+    public boolean handle(World world, BlockPos pos, EnumFacing direction, ItemStack stack, EntityPlayer player, IStripesActivator activator) {
+        EnumDyeColor pipeWireColor = EnumDyeColor.byMetadata(stack.getMetadata());
+
+        for (int i = PIPES_TO_TRY; i > 0; i--) {
+            pos = pos.offset(direction.getOpposite());
+
+            TileEntity tile = world.getTileEntity(pos);
+            if (tile != null && tile.hasCapability(PipeApi.CAP_PIPE_HOLDER, null)) {
+                IPipeHolder pipeHolder = tile.getCapability(PipeApi.CAP_PIPE_HOLDER, null);
+
+                /*
```

Comment:
I need feedback on how to implement this ;)

Reply:
I suppose this is a bit more complicated :(

Ideally I think this would place wires down adjacent to where they are on the previous pipe, so if a pipe is going NORTH then it should:

1) Check to see if the current pipe has any wire of the given colour
   - If yes, then try to continue the line of pipe wire forwards or backwards along that direction
   - If no:
      1) Find the closest wire with the same colour on the pipe before it.
               if found: continue that wire along the north (backwards or forewards)
               if no: add the wire on the north side of *this* pipe to the best possible slot (or random)

By best possible slot I mean a slot without any other colours of wire going in that direction on either this pipe or the one before.

That's just a test instruction list though, it might not actually work that way in code.